### DISCUSSION TITLE
Quest: Assist the Javalentians

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#8] [Announcements]
**Quest**: Assist the Javalentians.

**Details**:

Word has reached the Obsidian TTRPG Community that the people of Javalent have begun a focused effort on reworking [Obsidian Fantasy Calendar](https://github.com/fantasycalendar/obsidian-fantasy-calendar). 

We have confirmed this information is accurate. In confirmation of this we have been asked to provide as much assistance as possible in protecting the people of Javalent from the constant invasions of bugs, feature-monsters, and mimic threats alike. 

If you are able to assist, our contact is @valentine195 and you are encouraged to show up on the front lines (where you will receive the Pull-Requestor 5000, a sweet, sweet gun.).

**Rewards**:

- 1 Pull-Requester 5000
- Notoriety
- A statue
- 500g
- Pocket Lint

---

## Repo List & Issues

**Admonitions**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/admonitions?color=red&style=flat-square)

**Dice Roller**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/dice-roller?color=pink&style=flat-square)

**Fantasy Statblocks**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/fantasy-statblocks?color=blue&style=flat-square)

**Initiative Tracker**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/initiative-tracker?color=yellow&style=flat-square)

**Markdown Attributes**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/markdown-attributes?color=orange&style=flat-square)

**Obsidian Leaflet**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/obsidian-leaflet?color=green&style=flat-square)

**Second Window**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/second-window?color=purple&style=flat-square)

**Settings Search**
![GitHub issues](https://img.shields.io/github/issues-raw/javalent/settings-search?color=grey&style=flat-square)

**Comment by sigrunixia** [comment]
**Translation**:
We want Javalent to focus on this long-time coming project. As such, anyone who can help knock down the PR's, issues, and such in his repositories, please do so. There is work for all skill levels!

**Reply by Yilmas** [reply]
Think it would be great if you listed the repo's in question. Don't know if valentine has enabled issue images... I can't remember wth they are called. That one liner that auto updates an image which shows open issues, recent release builds, etc.

**Reply by Yilmas** [reply]
These, but for github...
https://shields.io/category/issue-tracking

**Reply by valentine195** [reply]
There’s nothing to enable for those, you just build the link. E.g. this is for leaflet:

https://img.shields.io/github/issues/javalent/obsidian-leaflet

then you can just embed it

![](https://img.shields.io/github/issues/javalent/obsidian-leaflet)

**Reply by sigrunixia** [reply]
Good Idea. I'll get those added into the post.

**Reply by sigrunixia** [reply]
And Added
